test/k8sT/Services.go

```diff
@@ -2815,6 +2815,45 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 					testNodePort(true, false, false, 0) // no need to test from outside, as testDSR did it
 				})
 
+				// GKE COS image does not support sctp module.
+				SkipContextIf(func() bool {
+					return !(helpers.RunsWithKubeProxyReplacement() && helpers.DoesNotRunOnGKE())
```

Comment:
Nit: I think it's a bit more explicit the other way around:
```suggestion
					return helpers.RunsWithoutKubeProxyReplacement() || helpers.RunsOnGKE()
```

Reply:
Absolutely!!